README.md

```diff
@@ -1 +1,65 @@
-# community
\ No newline at end of file
+# ROS 2 Security Working Group
+The Security Working Group's mission is to advocate for and implement security features within ROS 2.  The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.
+
+This document outlines the governance of the ROS 2 Security Working Group.  Updates to this document will be handled in the same manner as other project updates governed by the Working Group.
+
+## Subprojects
+A significant portion of the Security Working Group's mission is achieved by maintaining ROS subprojects.  The following subprojects are owned by this Working Group:
```

Comment:
```suggestion
A significant portion of the Security Working Group's mission is achieved by maintaining ROS subprojects. 
The following subprojects are owned by this Working Group:
```

Reply:
Removed all double-spacing after periods as they don't render in md anyway.